include/util/mercator.hpp

```diff
@@ -1,17 +1,25 @@
 #ifndef MERCATOR_HPP
 #define MERCATOR_HPP
 
+#include <cmath>
+
 namespace osrm
 {
 namespace util
 {
+namespace mercator
+{
 
-struct mercator
+inline double yToLat(const double value) noexcept
 {
-    static double y2lat(const double value) noexcept;
+    return 180. * M_1_PI * (2. * std::atan(std::exp(value * M_PI / 180.)) - M_PI_2);
+}
 
-    static double lat2y(const double latitude) noexcept;
-};
+inline double latToY(const double latitude) noexcept
+{
+    return 180. * M_1_PI * std::log(std::tan(M_PI_4 + latitude * (M_PI / 180.) / 2.));
```

Comment:
Yes, but does not make a difference since both sub expressions are calculated at compile time and
```python
>>> (math.pi / 180) / 2.
0.008726646259971648
>>> math.pi / 360.
0.008726646259971648
```
so we do not even get better precision out of it.